mlflow/models/python_api.py

```diff
@@ -75,6 +76,10 @@ def build_docker(
             The version of installed mlflow will be the same as the one used to invoke this command.
 
         mlflow_home: Path to local clone of MLflow project. Use for development only.
+
+        base_image: Base image for the Docker image. If not specified, the default image is either
+            UBUNTU_BASE_IMAGE = "ubuntu:20.04" or PYTHON_SLIM_BASE_IMAGE = "python:{version}-slim"
+            Note: If custom image is used, there are no guarantees that the image will work.
```

Comment:
We can say that if base image is built on top of Ubuntu image, then it will have the best compatibility.